Mathlib/Algebra/Algebra/Defs.lean

```diff
@@ -253,6 +253,34 @@ theorem RingHom.algebraMap_toAlgebra {R S} [CommSemiring R] [CommSemiring S] (i
   rfl
 #align ring_hom.algebra_map_to_algebra RingHom.algebraMap_toAlgebra
 
+lemma RingHom.isScalarTower_toAlgebra_comp {A B C : Type*}
```

Comment:
This is just `IsScalarTower.of_algebraMap_eq'`

Reply:
Thanks!